test/integration/render-tests/within/paint-icon/style.json

```diff
@@ -0,0 +1,104 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 150,
```

Comment:
I think viewport size could be decreased to 64x64?

Reply:
changed.